__init__.py

```diff
@@ -0,0 +1,44 @@
+import fiftyone.operators as foo
+import fiftyone.operators.types as types
+
+from fiftyone.docs_search.query_index import query_index
+
+
+class SearchDocs(foo.DynamicOperator):
+  def __init__(self):
+    super().__init__(
+      "@voxel51/search-docs",
+      "Search the FiftyOne docs",
+    )
+  
+  def resolve_input(self, ctx):
+    inputs = types.Object()
+    inputs.str("query", label="Query", required=True)
+    return types.Property(inputs)
```

Comment:
resolve_input returns a `Property` to allow for customizing the entire form view the `property.view`. This is because the entire form is actually defined by a `Property` of the operators definition (which is a `types.Object`).

In the operator type system, base types like object do not have ui metadata associated with them, since they should be re-usable and generic (in both senses of the word eg. `List(String)` - List is a "Generic" type aka template type.)